lib/releases.js

```diff
@@ -57,7 +57,7 @@ const findReleases = async ({
     ? commitishFilteredReleases.filter((r) => r.tag_name.startsWith(tagPrefix))
     : commitishFilteredReleases
   const sortedPublishedReleases = sortReleases(
-    filteredReleases.filter((r) => !r.draft)
+    filteredReleases.filter((r) => !r.draft || !r.prerelease)
```

Comment:
I can't see how this could be anything but `&&`, unless it was `filterNot`

draft | prerelease | candidate
--- | --- | ---
false | false | true
true | false | true
false | true | true
true | true | false

Only in the case that what's being evaluated is a draft for a prerelease does it not factor in. Presumably we want to only consider that which are not draft AND not prerelease.

I am on mobile though, so perhaps missing context, but `&&` seems like it would work, you're saying it didn't?